### PR TITLE
Update Bridges Subtree

### PR DESCRIPTION
**Pull request by HCastano**
This PR is a follow up to #2515 where the `parity-bridges-common` repo was added a a Git
Subtree.

Unknown to us at the time, squash merging that PR ended up putting us in a situation
where we were unable up update the bridges subtree via `git subtree pull`. This is
because we were already working with a squashed version of the bridge history (since we
added the repo through `git subtree add ... --squash`).

Note that this time we should **not** squash merge this PR so that in the future we can
easily update the bridges subtree using `git subtree pull`

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -26,8 +26,10 @@ tempfile = "3.2.0"
 
 [workspace]
 members = [
-	"bridges/primitives/kusama",
-	"bridges/primitives/polkadot",
+    "bridges/primitives/chain-kusama",
```
Comment (ordian) [inline Cargo.toml:29]: nice attempt to sneak in spaces ;)

**Review by tomusdrw — APPROVED** [review]
LGTM! To anyone hitting the green button at the end: please make sure to **NOT squash-merge** this, but rather use regular merge as indicated in the description.

**Comment by ordian** [comment]
how are we supposed to merge it?
image

**Comment by HCastano** [comment]
> how are we supposed to merge it?
> image

So a squash-merge won't work for the above stated reasons, a rebase-merge won't work due to conflicts, and a regular merge is disabled.

So we either need someone with admin access to (temporarily) disable regular merges, or merge manually and push to `master`

**Pull request merged**